src/pages/hardware/protocolReference/rs232Usb.tsx

```diff
@@ -53,7 +54,7 @@ const integrationProcess: ReactNode = (
     <div className="paragraph">
         <p>The following process should be followed when integrating a third-party device with the GO device using our Third-Party
             Data Protocol:</p>
-        <h2>Contact Solutions Engineering</h2>
+        <h2>Contact solutions engineering</h2>
```

Comment:
```suggestion
        <h2>Contact Solutions Engineering</h2>
```
Per the capitalization rules, team names have to be capitalized